### PR TITLE
Create helm chart for Friendbot

### PR DESCRIPTION
**Pull request by mwtzzz**
What
Create a helm chart for friendbot.

Why
We want to provide easy-to-use helm charts so the public can deploy a whole private network at the push of a button.

Issue addressed by this PR
N/A

### CODE REVIEW

File: charts/friendbot/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+name: friendbot
+description: This chart will deploy friendbot to your private network
+version: 0.0.1
+appVersion: "6fbbdc47"
+description: Stellar Friendbot Helm Chart
+maintainers: 
+  - name: Stellar Development Foundation Ops team
```
Comment (satyamz) [inline charts/friendbot/Chart.yaml:8]: Remove ops team.

File: charts/friendbot/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+{{- define "common.friendbotImage" -}}
+{{ .Values.global.image.friendbot.registry }}/{{ .Values.global.image.friendbot.repository }}{{ .Values.global.image.friendbot.name }}:{{ .Values.global.image.friendbot.tag | default .Chart.AppVersion }}
+{{- end -}}
+
+{{- define "core.config" -}}
+{{- if eq .Values.global.network "testnet" -}}
+{{- template "core.testnetConfig" }}
+{{- else if eq .Values.global.network "pubnet" -}}
+{{- template "core.pubnetConfig" }}
```
Comment (satyamz) [inline charts/friendbot/templates/_helpers.tpl:30]: No need to add pubnet.

File: charts/friendbot/templates/friendbot01.yaml [hunk 3]
```diff
@@ -0,0 +1,141 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "common.fullname" . }}
+  {{- if .Release.Namespace }}
+  namespace: {{ .Release.Namespace }}
+  {{- else }}
+  namespace: sandbox
```
Comment (satyamz) [inline charts/friendbot/templates/friendbot01.yaml:9]: change this to default or simply remove this line.

**Review by jacekn — APPROVED** [review]

**Review by satyamz — APPROVED** [review]
LGTM

**Pull request merged**